### PR TITLE
Update rpi.mdx

### PR DESCRIPTION
**Pull request by Owen-sz**
Update directions for zstd compressed images.

### CODE REVIEW

File: src/content/docs/en/installation/rpi.mdx [hunk 1]
```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
```
Comment (nothingneko) [inline src/content/docs/en/installation/rpi.mdx:38]: unzstd isn’t included by default, maybe suggest the zstd -d method

File: src/content/docs/en/installation/rpi.mdx [hunk 2]
```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
...
+- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a .zip
```
Comment (nothingneko) [inline src/content/docs/en/installation/rpi.mdx:39]:
```suggestion
- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a Zip file.
```

File: src/content/docs/en/installation/rpi.mdx [hunk 3]
```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
...
+- On Windows 10, use 7zip
+- On macOS, install `zstd` from homebrew, then run the above command for Linux
```
Comment (nothingneko) [inline src/content/docs/en/installation/rpi.mdx:41]:
```suggestion
- On macOS, install `zstd` from Homebrew, then run the above command for Linux
```
also maybe just say the command again lol, then you can include the install command

**Review by nothingneko — APPROVED** [review]

**Comment by nothingneko** [comment]
meant to request changes, please resolve these

File: src/content/docs/en/installation/rpi.mdx (hunk 3 above)
Comment (Owen-sz) [reply src/content/docs/en/installation/rpi.mdx:41]: Good idea

File: src/content/docs/en/installation/rpi.mdx (hunk 1 above)
Comment (Owen-sz) [reply src/content/docs/en/installation/rpi.mdx:38]: It's a part of zstd

File: src/content/docs/en/installation/rpi.mdx [hunk 4]
```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
+- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a Zip file.
+- On Windows 10, use 7zip
```
Comment (nothingneko) [inline src/content/docs/en/installation/rpi.mdx:40]: WinZip and WinRar also work

File: src/content/docs/en/installation/rpi.mdx [hunk 5]
```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
...
+- On macOS, install `zstd` from Homebrew (`brew install zstd`), then `unzstd *FILENAME.img.zst*`.
+
 Grab a USB stick! Use a flasher program to flash Ultramarine to your storage device. [balenaEtcher](https://www.balena.io/etcher/) is an easy cross-platform choice.
```
Comment (nothingneko) [inline src/content/docs/en/installation/rpi.mdx:43]: USB may be too specific here

**Review by nothingneko — CHANGES_REQUESTED** [review]

File: src/content/docs/en/installation/rpi.mdx (hunk 4 above)
Comment (Owen-sz) [reply src/content/docs/en/installation/rpi.mdx:40]: I didn't want to include more then one option (for example getting zstd on MacPorts), but I can

File: src/content/docs/en/installation/rpi.mdx (hunk 5 above)
Comment (Owen-sz) [reply src/content/docs/en/installation/rpi.mdx:43]: Can fix that then lol

Comment (Owen-sz) [reply src/content/docs/en/installation/rpi.mdx]: Actually, storage devices are covered earlier, thoughts on just removing 'Grab a USB stick!'?

File: src/content/docs/en/installation/rpi.mdx (hunk 4 above)
Comment (nothingneko) [reply src/content/docs/en/installation/rpi.mdx:40]: at least add winrar because it's very very popular

**Review by nothingneko — APPROVED** [review]

**Pull request merged**